src/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
-
+import 'style/main.scss';
 import App from './App';
```

Comment:
Отделяем все блоки импортов одной пустой строкой, стили в самом конце

```suggestion

import App from './App';

import 'style/main.scss';
```

Reply:
fixed